Library/Homebrew/os/mac/keg.rb

```diff
@@ -1,7 +1,14 @@
-# typed: true # rubocop:todo Sorbet/StrictSigil
+# typed: strict
 # frozen_string_literal: true
 
 class Keg
+  sig { params(path: Pathname).void }
+  def initialize(path)
+    super
+
+    @require_relocation = T.let(nil, T.nilable(T::Boolean))
```

Comment:
```suggestion
    @require_relocation = T.let(false, T::Boolean)
```
unless you specifically need `nil` and `false` to be different states